### DISCUSSION TITLE
[3일차-윤경] json이란?

### DISCUSSION THREAD

**Discussion by skyhanull** [zzangCS/zzangInterview#15] [프론트 면접 질문]
json이란 무엇일까요

**Comment by ez1n** [comment]
**JSON**
JavaScript Object Notation의 약자로 데이터를 저장하거나 전소할 때 많이 사용되는 경량의 데이터 교환 방식

- 자바스크립트 객체를 만들 때 사용하는 표현식
- key, value로 이뤄져 있으며 ""(쌍따옴표)를 이용하여 표기
- 최근에는 XML을 대체하여 데이터 전송 등에 많이 사용함

**Comment by smarfy99** [comment]
Javascript Object Notation으로 데이터를 주고 받기 위한 형식으로, 용량이 작은 것이 특징이다. 자바스크립트 객체 형식을 가졌으며, 표현법이 간단하다.
